tests/test_update_ssh_keys.rs

```diff
@@ -0,0 +1,495 @@
+        add_one_ssh_key(
+            "valid2",            // pubkeyname
+            "one",               // testname
+            false,               // is_force
+            true,                // is_replace
+            false,               // is_stdin
+            true,                // is_expected_success
+            ["valid2"].to_vec(), // assert_keys
+        );
+    }
+
+    #[test]
+    fn test007_no_replace() {
+        // "update-ssh-keys --delete one valid2.pub"
+        del_one_ssh_key("valid2", "one");
+
+        // "update-ssh-keys --add one valid1.pub"
+        test004_add_one_file();
+
+        // "update-ssh-keys --no-replace --add one valid2.pub"
+        add_one_ssh_key(
+            "valid2",            // pubkeyname
+            "one",               // testname
+            false,               // is_force
+            false,               // is_replace
+            false,               // is_stdin
+            false,               // is_expected_success
+            ["valid1"].to_vec(), // assert_keys
+        );
+
+        // "update-ssh-keys --no-replace --add-force one valid2.pub"
+        add_one_ssh_key(
+            "valid2",            // pubkeyname
+            "one",               // testname
+            true,                // is_force
+            false,               // is_replace
+            false,               // is_stdin
+            false,               // is_expected_success
+            ["valid1"].to_vec(), // assert_keys
+        );
+    }
+
+    #[test]
+    fn test008_add_two() {
+        // "update-ssh-keys --add one valid1.pub"
+        test004_add_one_file();
+
+        // "update-ssh-keys --add two valid2.pub"
+        add_one_ssh_key(
+            "valid2",                      // pubkeyname
+            "two",                         // testname
+            false,                         // is_force
+            true,                          // is_replace
+            false,                         // is_stdin
+            true,                          // is_expected_success
+            ["valid1", "valid2"].to_vec(), // assert_keys
+        );
+    }
+
+    #[test]
+    fn test009_del_one() {
+        // "update-ssh-keys --add one valid1.pub"
+        test004_add_one_file();
+
+        // "update-ssh-keys --delete one valid1.pub"
+        del_one_ssh_key("valid1", "one");
+    }
+
+    #[test]
+    fn test010_del_two() {
+        // "update-ssh-keys --delete one valid1.pub"
+        // "update-ssh-keys --delete two valid2.pub"
+        del_one_ssh_key("valid1", "one");
+        del_one_ssh_key("valid2", "two");
+
+        // "update-ssh-keys --add one valid1.pub"
+        // "update-ssh-keys --add two valid2.pub"
+        test008_add_two();
+
+        // "update-ssh-keys --delete two valid2.pub"
+        del_one_ssh_key("valid2", "two");
+    }
+
+    #[test]
+    #[ignore] // ignore the test, as test_disable will be anyway called by test_enable() below.
```

Comment:
Ignoring the test doesn't look like the right solution, I suggest you try to get it working after fixing the test setup.

Reply:
no ignore any more